index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
```

Comment:
use to styles in separate css file for better and efficient use don't use inline css.